receipt/scontrino0.txt

```diff
@@ -0,0 +1,17 @@
+
++ -------------------------------------------------- +
+ICTS21-23.096 - Cart 2860
+gio 14 apr 2022
+* -------------------------------------------------- *
+	[120193]	Alpi		22.1			
+	[812302]	Firenze		9.99			
+	[912301]	Roma		9.99			
+	[912303]	Pisa		9.99			
+* -------------------------------------------------- *
+	Totale:					52.07
++ -------------------------------------------------- +
+	
+** -------------------------------------------------- **
+	Winston Wolf ha un credito residuo di 10.04
```

Comment:
Perché tutte le soluzioni hanno una doppia riga (una giusta e una sbagliata) ?

Guarda riga 27 e 29 di cart.mjs

Reply:
Questo purtroppo, come ho scritto nel commento all'interno del codice, non riesco a creare un metodo che richiami o un o l'altro in base alla presenza del promo code

Reply:
Va bene, ricordamelo in classe che lo vediamo insieme.